lib/handlers/hard-quote/handler.ts

```diff
@@ -109,7 +109,7 @@ export class QuoteHandler extends APIGLambdaHandler<
       cosignerData = getCosignerData(request, bestQuote, orderType);
       log.info({ bestQuote: bestQuote }, 'bestQuote');
     } else {
-      cosignerData = getDefaultCosignerData(request, orderType);
+      cosignerData = await getDefaultCosignerData(request, orderType, provider);
```

Comment:
I think we should do something similar to URA:
`const provider = chainIdRpcMap.get(requestBody.tokenInChainId);`

I know it's only used for arbitrum today but I worry that some dev in the future will call `provider.getBlockNumber()` expecting it to the be the provider of the current chain and not get any kind of error message.